schutzbot/prepare-rhel-internal.sh

```diff
@@ -64,7 +64,7 @@ cp rhel-8.json rhel-8-beta.json
 
 JOB_NAME="${JOB_NAME:-${CI_JOB_ID}}"
 # Do not create tests repo if it's provided from ENV
-if [ -z "$REPO_URL" ]; then
+if [ -z "${REPO_URL+x}" ]; then
```

Comment:
I don't understand the `+x` part.

Reply:
I got this from here https://stackoverflow.com/a/13864829  I've initially set this up so that we can use custom URL to a scratch build for the nightly testing pipeline. It worked fine until I've removed the variable because we no longer need it and want to use the rpm that's in the nightly compose and since we use `set -u` it fails on this as can be seen here: https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1518775766#L523